sdk/nodejs/runtime/stack.ts

```diff
@@ -233,6 +233,16 @@ export function registerStackTransform(t: ResourceTransform) {
     callbacks.registerStackTransform(t);
 }
 
+/**
+ * Add a transformation to all future resources constructed in this Pulumi stack.
+ *
+ * @deprecated Use `registerResourceTransform` instead.
+ */
+export function registerStackTransform(t: ResourceTransform) {
```

Comment:
Are these functions return type `void`? Guessing yes? If so, worth adding that while we're here, or is that a terrible/non-backwards-compatible change?

Reply:
Yes, I think so.  I'm not sure if that's backwards compatible tbh.  I can definitely add it to the new `registerResourceTransform` since there's no backwards compatibility concerns there.